polygon-id/id3/id3.json

```diff
@@ -1804,8 +1804,609 @@
               "type": "object",
               "properties": {
                 "citizenship": {
-                  "title": "Citizenship",
-                  "description": "The country of citizenship of the credential subject - ISO 3166-1 alpha-3 country code (https://www.iban.com/country-codes)",
+                  "title": "Citizenship - ISO 3166-1 alpha-3 country code (https://www.iban.com/country-codes)",
```

Comment:
do you really want the title to be the long title instead of Citizenship?

Reply:
let's fix it

Reply:
fixed